wesnoth-units.po

```diff
@@ -5807,7 +5863,7 @@ msgstr ""
 #: data/core/units/nagas/Ringcaster.cfg:42 data/core/units/nagas/Zephyr.cfg:50
 #: data/core/units/nagas/Zephyr.cfg:59
 msgid "chakram"
-msgstr ""
+msgstr "čakram"
```

Comment:
Přiznám se, že jsem toto PR ještě pořádně neprocházel a asi mu věnuju ještě jedno kolo review, ale toto mě už v emailu trklo. Když jsem ještě hrál online MMORPG hry jako 4story a Metin, tak překlad slova `chakram` neexistoval a slovo bylo počeštěno. Já vím, že slovo `čakram` existuje a je mu věnovaná i [stránka české wikipedie](https://cs.wikipedia.org/wiki/Čakram), ale stejně bych byl pro zachování slova `chakram` a jeho počeštění. Nechť rozhodně @celerini .

```suggestion
msgstr "chakram"
```

Reply:
No ono je to uvedené i jinde, než na wikipedii. Někdy tedy ve tvaru čakrom.
Každopádně mi přijde že chak na začátku slova nejde češtinářsky do pusy. A také, že v překladech jsou už i jiné výskyty, kdy se CH přetaktovalo na Č.

Reply:
tohle slovo neznám, tak nechci být papežštější...

vyhledávání site:cz dává podle počtu výsledků za pravdu Martinovi, ale já bych tento autorský vklad nechal tak, jak je (čakram)